### PR TITLE
Added autofill function to project meta comparison and delete project functionality

### PR DESCRIPTION
**Pull request by sofiahag**
First version. I will add the default graph and such later.

Tell me what you think about the look! 
I just added the "x", not sure if that's intuitive enough? 
The plot is loaded by selecting the x- and y-axis values. Then we remove the project by clicking the badge and re-load the plot by re-selecting x- and y axes.

I tried to break it but couldn't, but feel free to try 😄

### CODE REVIEW

**Comment by sofiahag** [comment]
Up on stage!

File: run_dir/static/css/status_b5.css [hunk 1]
```diff
@@ -83,6 +83,15 @@ tfoot {
 .tt-suggestion p { margin: 0; }
 .tt-hint { visibility:hidden; }
 
+/* Project Meta Comparison search badges */
+
+#badges .badge {
+  font-size: 1em;
+}
+
+#badges .badge:hover {
```
Comment (aanil) [inline run_dir/static/css/status_b5.css:92]: Maybe a more specific name than just `badges`?

Comment (sofiahag) [reply run_dir/static/css/status_b5.css:92]: Sure, I was thinking of it, but it shouldn't be too long 🤔 

`#pid_badges`? `#pid_del_badges`?

Comment (aanil) [reply run_dir/static/css/status_b5.css:92]: More descriptive the better. It doesn't matter if its a bit longer, as long as its easily understandable.

File: run_dir/static/js/proj_meta_compare.js [hunk 2]
```diff
@@ -29,15 +31,52 @@ yLogAxis = 'linear';
...
+      highlight: true,
+    }, {
+      name: 'project-search',
+      displayKey: 'name',
+      source: projectSearch.ttAdapter(),
+      templates: {
+          empty: '<div class="empty-message">No projects found</div>'
+      }
+    }).bind('typeahead:selected', function(obj, datum, name) {
+        var proj_id = datum.url.split('/')[2];
```
Comment (aanil) [inline run_dir/static/js/proj_meta_compare.js:55]: Use `let` instead of `var`

File: run_dir/static/js/proj_meta_compare.js [hunk 3]
```diff
@@ -29,15 +31,52 @@ yLogAxis = 'linear';
...
+        $('#projects_meta_input').val('');
+        $('#badges').append('<button class="btn badge rounded-pill bg-secondary mx-1" id="' + proj_id +  '">' + proj_id + ' x' + '</button>');
+        id_tosave.push(proj_id);
+        $("#badges > button").on("click", function() {
+            $('#projects_meta_input').val('');
+            var but_id = $("#"+$(this).attr('id'));
```
Comment (aanil) [inline run_dir/static/js/proj_meta_compare.js:61]: `let` here too

File: run_dir/static/css/status_b5.css (hunk 1 above)
Comment (sofiahag) [reply run_dir/static/css/status_b5.css:92]: Then I'll go with `#del_pid_badges` 👍🏻

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**